src/schema-association-service.ts

```diff
@@ -177,6 +182,13 @@ async function autoDetectSchema(
     } else {
         logger.log(`${workspaceFolder.name} has no remote URL or is not an Azure repo`, 'SchemaDetection');
 
+        const doNotAskAgainSelectOrg = context.globalState.get<boolean>('doNotAskAgainSelectOrg');
+        logger.log(`doNotAskAgainSelectOrg status: ${doNotAskAgainSelectOrg}`);
```

Comment:
I think it'd make more sense to put the log inside the if and change the message to something like "Exiting early (do not ask again)"